qobuz_importer.user.js

```diff
@@ -42,10 +42,10 @@ function parseRelease(data) {
 
   // Release information global to all Beatport releases
   release.packaging = 'None';
-  release.country = "";
+  release.barcode = data.upc;
+  release.country = "XW";
   if (i18n_global && i18n_global.zone) {
-    if (i18n_global.zone == 'GB') release.country = 'UK';
```

Comment:
I think this line should stay, it converts 'GB' used by Qobuz to 'UK' used by MB

Reply:
That is to say, I use the site in en-gb and GB is valid for seeding, but UK isn't.